src/api/http_test.go

```diff
@@ -26,11 +26,12 @@ var allAPISetsEnabled = map[string]struct{}{
 
 func defaultMuxConfig() muxConfig {
 	return muxConfig{
-		host:           configuredHost,
-		appLoc:         ".",
-		disableCSRF:    true,
-		disableCSP:     true,
-		enabledAPISets: allAPISetsEnabled,
+		host:               configuredHost,
+		appLoc:             ".",
+		disableCSRF:        true,
+		disableHeadercheck: true,
```

Comment:
Is it necessary to have it true by default? Don't the tests already cope with the header check